src/verify/VerificationStartScreen.tsx

```diff
@@ -124,9 +125,20 @@ function VerificationStartScreen({
           />
         ),
       headerLeft: () => route.params?.hideOnboardingStep && <BackButton />,
+      // Disable iOS back during onboarding
+      gestureEnabled: route.params?.hideOnboardingStep ? false : true,
```

Comment:
this causes the same issue as i mentioned before, where if a user isn't launching this screen from onboarding -> they select the phone number country -> the `hideOnboardingStep` becomes true and we display "step 4 /4". this is not a deal breaker i guess

Reply:
Fixed in 00b81317f5c7ef19745a55f8674f7564ffca6d79!